Puppetfile

```diff
@@ -17,5 +17,5 @@ mod 'theforeman/foreman',              '~> 23.1.0'
 mod 'theforeman/foreman_proxy',        '~> 25.1.0'
 mod 'theforeman/puppet',               '~> 18.0.0'
 mod 'katello/foreman_proxy_content',   '~> 25.0.0'
-mod 'katello/certs',                   '~> 16.0.2'
+mod 'katello/certs',                   '~> 16.1.0'
```

Comment:
should this enforce `>= 16.1.1` given 16.1.0 is "broken"?

Reply:
Technically it should be sufficient to enforce it via the lock file, but I guess it doesn't hurt.